packages/rancher-alerting-drivers/charts/templates/hardened.yaml

```diff
@@ -83,7 +82,7 @@ spec:
   hostIPC: false
   hostPID: false
   runAsUser:
-    rule: 'MustRunAsNonRoot'
+    rule: 'RunAsAny'
   seLinux:
     rule: 'RunAsAny'
```

Comment:
we need RunAsAny because the pod needs to run as root to be able to use inClusterConfig